### PR TITLE
Add newrelic integration

### PR DESCRIPTION
**Pull request by nijamashruwala**

### CODE REVIEW

**Comment by nijamashruwala** [comment]
Integrate new relic monitoring @jontg @christydennison @nari-m

**Comment by christydennison** [comment]
@nijamashruwala Do you know about the https://github.com/relateiq/soa-boilerplate repo? Generally for soa updates we first update the boilerplate and then run the to-template.sh script in the boilerplate to copy over to soa, otherwise your changes might get copied over next time someone else uses the boilerplate script. You can see more information in the README for the repo.

File: soa-archetype/src/main/resources/archetype-resources/__rootArtifactId__-server/start.sh [hunk 1]
```diff
@@ -17,5 +20,75 @@ for riqdir in ${dbr}RIQ_DIRS[@]}; do
...
+# jvm options default for staging/prod
+JVM_OPTIONS=""
+# common jvm options regardless test/staging/prod
+JVM_COMMON_OPTIONS=""
+# debug options for non-prod
+JVM_DEBUG_OPTIONS=""
+# set debug on for non-prod envs
+if [ "${APP_ENVIRONMENT}" = "staging" ]; then
+    # use jvm-debug_options in non-prod
+    JVM_OPTIONS="${JVM_OPTIONS} ${JVM_DEBUG_OPTIONS}"
```
Comment (christydennison) [inline soa-archetype/src/main/resources/archetype-resources/__rootArtifactId__-server/start.sh]: This won't work once the generator runs. Please put these changes in the boilerplate first and run the script.

Comment (nijamashruwala) [inline soa-archetype/src/main/resources/archetype-resources/__rootArtifactId__-server/start.sh]: Thanks, please ignore this change

**Pull request closed without merging**